authorizerd.go

```diff
@@ -350,8 +352,21 @@ func (a *authorizer) verify(ctx context.Context, m mode, tok, act, res string, c
 		return errors.Wrap(ErrInvalidParameters, "empty action / resource")
 	}
 
+	var certThumbprint string
 	// check if exists in verification success cache
-	_, isCacheHit := a.cache.Get(tok + act + res)
+	if cert != nil {
+		ct := sha256.Sum256(cert.Raw)
+		certThumbprint = hex.EncodeToString(ct[:])
+		if _, ok := a.cache.Get(tok + act + res + certThumbprint); ok {
```

Comment:
1. seems better to refactor as a `getCacheKey` func.
2. latency (use thumbprint) V.S. memory (use raw)